### ISSUE TITLE
Broken on install (new note/folder don't work)

### ISSUE THREAD

**Issue by ijeppsen** [ryanmcalister/unotes#8]
After installing on vscode 1.33.1  on Mac OSX I get the following when clicking the new note,folder buttons in unote.
`command 'unotes.addNote' not found 
command 'unotes.addFolder' not found 
`

**Comment by ijeppsen** [comment]
Note, this seems to be happening because I didn't 'open a folder' first.  Might be worth adding that to the readme

**Comment by Dynalt-glj1** [comment]
Running VS Code on Windows 7, I get an additional message:

"There is no data provider registered that can provide view data."

**Comment by ryanmcalister** [comment]
Yes, Unotes currently requires a workspace folder to be open before use. I added a simple warning for now in v.1.0.5 and a README note. I plan to handle this better in the future. 
Thanks for the report!

**Comment by vconerd** [comment]
Hi, Ryan. Congrats for your excelent work on this plugin!
I suggest you to give us a config option where we can set a default markdown creation folder, by sample: /home/user/Projects, or /home/user/TODO list, or /home/user/Schedule, or /home/user/Pending Tasks, or simply /home/user/UNotes.
